spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/post/SendResponseFilter.java

```diff
@@ -172,27 +173,25 @@ else if (context.getResponseGZipped() && isGzipRequested) {
 		}
 		finally {
 			try {
-				if (is != null) {
```

Comment:
Should we not be closing the `InputStream` as well?  Or does doing that cause the issue you were trying to solve?  I just want to make sure we aren't introducing a memory leak of any kind...